### ISSUE TITLE
Switch user wheel usability on iphones with FaceID

### ISSUE THREAD

**Issue by bertez** [DagAgren/toot#574]
The user switch wheel is a pretty cool feature that makes this app unique, that said, most of the time when I use the wheel pulling it down the Reachability gesture is triggered (https://support.apple.com/guide/iphone/reachability-iph145eba8e9/ios) and if I pull it up the phone go to switch app mode

**Comment by alexbayleaf** [comment]
Yeah, I haven’t been able to get it to work properly at all because of that.
